test/Interop/Cxx/value-witness-table/Inputs/basic.h

```diff
@@ -0,0 +1,36 @@
+#ifndef TEST_INTEROP_CXX_DESTRUCTOR_INPUTS_BASIC_H
+#define TEST_INTEROP_CXX_DESTRUCTOR_INPUTS_BASIC_H
+
+struct HasUserProvidedDestructor {
+  int *value;
+  ~HasUserProvidedDestructor() { *value = 42; }
+};
+
+struct EmptyDestructor {
+  HasUserProvidedDestructor box;
+  ~EmptyDestructor() { /* empty */
+  }
+};
+
+struct HasNonTrivialImplicitDestructor {
+  HasUserProvidedDestructor box;
```

Comment:
This is a minor point, but I'd prefer if we didn't use the word "box" to describe by-value wrapping. "Box" in Swift (and other languages) has a strong association with the value being located on the heap, and its lifetime being managed dynamically.

Reply:
Good to know. I'll remove references to "box" in this PR and keep them out of future PRs (unless appropriate).